requirements.txt

```diff
@@ -1,3 +1,4 @@
 emoji
 tree-sitter
-semgrep
\ No newline at end of file
+tree-sitter-c
```

Comment:
Given the proclivity of tree-sitter API updates, maybe it's worth pinning these like you said?

When I tested your PR I had

```
tree-sitter==0.23.2
tree-sitter-c==0.23.4
```